javascript/ql/src/LanguageFeatures/ThisBeforeSuper.ql

```diff
@@ -59,4 +59,4 @@ where needsGuard(e, kind) and unguarded(e, ctor) and
         sc.getBinder() = ctor and
         sc.getEnclosingFunction() != ctor
       )
-select e, "The super constructor must be called before using '" + kind + "'."
\ No newline at end of file
+select ctor, "The super constructor must be called before using '$@'.", e, kind
```

Comment:
We generally want to avoid flagging multi-line entities, otherwise everything ends up looking very red on LGTM. You can use the `RestrictedLocations` library to only highlight the first line of the constructor.

Reply:
Done. Good catch.